x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks.test.tsx

```diff
@@ -80,7 +81,8 @@ describe('<Benchmarks />', () => {
     const error = new Error('message');
     renderBenchmarks(createReactQueryResponse({ status: 'error', error }));
 
-    expect(screen.getByText(error.message)).toBeInTheDocument();
+    expect(screen.getByText(ERROR_MESSAGE_TITLE)).toBeInTheDocument();
+    expect(screen.getByTestId(ERROR_STATE_TEST_SUBJECT)).toBeInTheDocument();
```

Comment:
Since the error message is not dynamic, only testing for the test subject would be enough for this case